pkgs/os-specific/linux/apparmor/default.nix

```diff
@@ -140,6 +153,15 @@ let
       # aa-notify checks its name and does not work named ".aa-notify-wrapped"
       mv $out/bin/aa-notify $out/bin/aa-notify-wrapped
       makeWrapper ${perl}/bin/perl $out/bin/aa-notify --set PERL5LIB ${libapparmor}/${perl.libPrefix} --add-flags $out/bin/aa-notify-wrapped
+
```

Comment:
aa-unconfined needs netstat at runtime

Reply:
Indeed `aa-unconfined` can't find `netstat` nor `ss` because it completely overwrites `PATH` :facepalm: https://gitlab.com/apparmor/apparmor/-/blob/master/utils/aa-unconfined#L89